src/websocket/index.ts

```diff
@@ -23,15 +23,30 @@ interface Request {
 
 // Add connection counter
 let activeConnections = 0
+const connectionsByIP = new Map<string, number>()
 
-export const onConnection = async (socket: WebSocket.WebSocket): Promise<void> => {
-  // Check max connections limit
-  if (activeConnections >= CONFIG.websocket.maxConnections) {
-    socket.close(1003, 'Server busy. Please try again later.')
+export const onConnection = async (socket: WebSocket.WebSocket, req: any): Promise<void> => {
```

Comment:
can we use specific type for `req` instead of any?

Reply:
included